src/main/java/unc/Duke.java

```diff
@@ -0,0 +1,198 @@
+package unc;
+
+import unc.task.Deadline;
+import unc.task.Event;
+import unc.task.Task;
+import unc.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+public class Duke {
+
+    private static List<Task> taskList;
+    private static File saveFile;
+
+    private enum validInput {
+        LIST,
+        MARK,
+        UNMARK,
+        DELETE,
+        TODO,
+        DEADLINE,
+        EVENT
+
+    }
+
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Welcome to UNC\n");
+
+        taskList = new ArrayList<>();
+        saveFile = new File("./data/save.txt");
+        try {
+            Scanner scanner1 = new Scanner(saveFile);
```

Comment:
Perhaps you could consider choosing a more specific name for the scanner variables since you have two in the same scope, rather than `scanner` and `scanner1`?